charts/duplicati/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: v2.0.5.1
 description: Store securely encrypted backups on cloud storage services!
 name: duplicati
-version: 2.0.1
+version: 2.0.2
```

```suggestion
version: 2.1.0
```

Even though this is a small feature addition, it's not a bug fix and therefore warrants a minor version bump.